src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementProjectService.java

```diff
@@ -362,21 +362,29 @@ public TeamMember addProjectTeamMember(final Integer projectId, final String use
     var dbProject = getProject(projects, project.getTender().getTenderCode());
     var projectPackageSummary = new ProjectPackageSummary();
     if (dbProject.isPresent()) {
+      var agreementNo = dbProject.get().getCaNumber();
       var dbEvent = getCurrentEvent(dbProject.get());
-
+      // TODO make single call instead of 2
+      try {
+        var agreementDetails = agreementsService.getAgreementDetails(agreementNo);
+        var lotDetails = agreementsService.getLotDetails(agreementNo, dbProject.get().getLotNumber());
+        projectPackageSummary.setAgreementName(agreementDetails.getName());
+        projectPackageSummary.setLotName(lotDetails.getName());
+      }catch (Exception e) {
```

Comment:
I would suggest that probably isn't going to be implemented any time soon so would remove this try-catch (would involve a significant change to the AS API.